tests/beman/exemplar/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ find_package(GTest REQUIRED)
 
 add_executable(beman.exemplar.tests.identity)
 target_sources(beman.exemplar.tests.identity PRIVATE identity.test.cpp)
+target_compile_features(beman.exemplar.tests.identity PRIVATE cxx_std_20)
```

Comment:
This violates [passive project requirement](https://github.com/bemanproject/beman/blob/5c0b8662f9e6ee7b995d5e14f4c690a6aac5ce7a/docs/BEMAN_STANDARD.md?plain=1#L344-L356) btw.

> `target_compile_features` (e.g., `cxx_std_20`) must not be used
because it modifies the compilation environment of dependent targets.